includes/emails/class-edd-email-tags.php

```diff
@@ -460,6 +460,15 @@ function edd_email_tag_download_list( $payment_id ) {
 						}
 					}
 				}
+			}else{
+
+				$no_downloads_message = apply_filters( 'edd_email_receipt_no_downloads_message', '', $item['id'], $price_id, $payment_id );
```

Comment:
Switch this to `edd_receipt_no_files_found_text` and I'm good with it, since that will make all plugins that remove this link from the purchase confirmation remove it from the email receipts without them needing to update.